tests/integration/crud/simple/chunks/test_chunks.py

```diff
@@ -0,0 +1,165 @@
+import os
+import random
+from itertools import chain
+from pathlib import Path
+
+import numpy as np
+import pytest
+from jina.executors.indexers import BaseIndexer
+
+from jina import Document
+from jina.flow import Flow
+
+TOP_K = 10
+
+@pytest.fixture
+def config(tmpdir):
+    random.seed(0)
+    np.random.seed(0)
+    os.environ['JINA_CRUD_CHUNKS'] = str(tmpdir)
+    os.environ['JINA_TOPK'] = '10'
+    yield
+    del os.environ['JINA_CRUD_CHUNKS']
+    del os.environ['JINA_TOPK']
+
+
+d_embedding = np.random.random([9])
+c_embedding = np.random.random([9])
+
+
+def document_generator(content_same, start, num_docs, num_chunks):
+    chunk_id = num_docs
+    for idx in range(start, num_docs):
+        with Document() as doc:
+            doc.id = idx
+            if content_same:
+                doc.tags['id'] = idx
+                doc.text = 'I have cats'
+                doc.embedding = d_embedding
+            else:
+                doc.text = f'I have {idx} cats'
+                doc.embedding = np.random.random([9])
+            for chunk_idx in range(num_chunks):
+                with Document() as chunk:
+                    chunk.id = chunk_id
+                    if content_same:
+                        chunk.tags['id'] = chunk_idx
+                        chunk.text = 'I have chunky cats'
+                        chunk.embedding = c_embedding
+                    else:
+                        chunk.tags['id'] = chunk_idx
+                        chunk.text = f'I have {chunk_idx} chunky cats'
+                        chunk.embedding = np.random.random([9])
+                chunk_id += 1
+                doc.chunks.append(chunk)
+        yield doc
+
+
+def validate_index_size(num_indexed_docs):
+    path = Path(os.environ['JINA_CRUD_CHUNKS'])
+    index_files = list(path.glob('*.bin'))
+    assert len(index_files) > 0
+    for index_file in index_files:
+        index = BaseIndexer.load(str(index_file))
+        assert index.size == num_indexed_docs
+
+
+@pytest.mark.parametrize('flow_file, content_same', [
+    ['flow_vector.yml', False],
+    ['flow_vector.yml', True]
+])
+def test_delete_vector(config, mocker, flow_file, content_same):
```

Comment:
I think I mentioned in the previous version, but again: `content_same` should be removed. It's unnecessary without the cache in front

Reply:
Yes, I'll make the changes now